### DISCUSSION TITLE
Parking access - publicly available private parkings [implemented in #3473]

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3429] [Feedback]
Implemented in #3473

For reference, this is how quest look like:

![screen09](https://user-images.githubusercontent.com/899988/138514750-7d502b49-9afb-488d-841c-ee539cb3556e.png)

People are a bit confused by private parking that are publicly available.

Maybe instead of 

> It is private.

something like

> General public is not allowed to use it.

would be better?

**Comment by andrewharvey** [comment]
I agree this is an issue. The wiki says "Only with individual permission" but I think "General public is not allowed access" is better.

"It's only for customers" is also unclear, as you could interpret a privately (ie run by a company instead of the government) operated car park, which anyone is allowed to pay to use, is only for "customers" of the car park. However the `customers` tag is meant to be only for customers of the associated facility.

Therefore I'd suggest the following:

> "Who may park here? Irrespective of any fees" ("Is it restricted" implies a yes/no answer, the first option is essential there is no restriction, and the last two are yes there are restrictions. All answers are irrespective of fees, not just the first option)
> 
> - Open to the general public
> - Not open to the general public
> - Open only to customers of the associated facility

**Reply by matkoniecz** [reply]
Here is my attempt to simplify language without losing anything:

> Who may park here? Parking may be free or paid.
> 
> Open to the general public 
> Not open to the general public
> Open only to customers of some object

**Reply by westnordost** [reply]
"some object" sounds weird.

**Reply by peternewman** [reply]
"Only open to customers of the business"?

Or "...of the associated business"

Pedantically there may be some edge cases that fall between the latter two, e.g. a youth group charity's centre, is that private or customers?

**Reply by matkoniecz** [reply]
> "Only open to customers of the business"?

This runs into problem when it is only for people visiting hospital/government office or in a front of school/military barracks etc.

**Reply by matkoniecz** [reply]
Maybe " Open only to customers of a specific place"?

Or "Open only to customers of given place"? Or " Open only to customers of some place"?

**Reply by westnordost** [reply]
"place" sounds ok. But why not "facility" though?

@andrewharvey suggested

> Open only to customers of the associated facility

**Reply by matkoniecz** [reply]
> But why not "facility" though?

Facility seems weird to me - like some technical/official/stilted word, and sounds like some factory. Calling shop "facility" seems weird to me.

(but I am NOT a native speaker)

**Reply by andrewharvey** [reply]
> Facility seems weird to me - like some technical/official/stilted word, and sounds like some factory. Calling shop "facility" seems weird to me.

"facility: a place, amenity, or piece of equipment provided for a particular purpose."

It comes down to translators though, who translate this into Canadian English, American English, Australian English, British English etc.

**Reply by matkoniecz** [reply]
What about

> Open only to people visiting a specific place (e.g. customers)

?

**Reply by matkoniecz** [reply]
I opened #3473

I someone (especially native speaker) feels that facility version is clearly superior - let me know and I will change it.

**Reply by smichel17** [reply]
I am somewhere in between.

> like some technical/official/stilted word, and sounds like some factory

In the context of movies and video games, I'd agree. But in everyday usage, I think it's more common. My family, at least, says "use the facilities" to mean "use the bathroom" (most often in the context of a long car ride).

However, I don't think I'd use *facilities* to describe something like a hiking trail, but there's definitely parking areas which are reserved (in theory) for people hiking there; as far as I understand, they should be tagged `access=customers`. So, I *think* I prefer the no-facility version.
